app/src/main/res/layout/main_fragment.xml

```diff
@@ -60,9 +62,11 @@
                 android:layout_marginStart="16dp"
                 android:layout_marginLeft="16dp"
                 android:layout_marginTop="8dp"
+                android:layout_marginBottom="16dp"
                 android:text="@string/location"
                 android:textColor="@color/colorPrimary"
                 android:textSize="18sp"
+                app:layout_constraintBottom_toTopOf="@+id/textView"
```

Comment:
While you're in here, rename textView to something more descriptive.  When you have several variables named "textView#", it's hard to tell which is which.